packages/api/webpack.config.js

```diff
@@ -80,7 +80,6 @@ module.exports = {
     }
   },
   externals: [
-    'aws-sdk',
     '@aws-sdk',
```

Comment:
Same question as above (same for all of the webpack configs). ELI5 please 😆

Reply:
Punting this conversation to https://github.com/nasa/cumulus/pull/3618#discussion_r1561655895